docs/pages/deployment/docker.rst

```diff
@@ -0,0 +1,131 @@
+.. _running-docker:
+
+Running on Docker
+#################
+
+This guide helps you to configure the Nuts node in Docker.
+To use the most recent release use ``nutsfoundation/nuts-node:latest``. For production environments it's advised to use a specific version.
+
+Examples
+********
+The examples assume the current working directory on the host contains the following structure:
+
+.. code-block:: none
+
+    .
+    ├── config
+    │   └── nuts.yaml
+    └── data
+
+The ``data`` directory must be owned by container user ``18081``. (``chown -R 18081:18081 ./data``)
+
+These examples use the following ``nuts.yaml`` configuration file:
+
+.. code-block:: yaml
+
+  strictmode: false
+  http:
+    internal:
+      address: :8081
+
+See the :ref:`configuration<nuts-node-config>` documentation how to setup the ``nuts.yaml`` to meet your needs.
+
+Docker Compose
+^^^^^^^^^^^^^^
+
+Copy the following YAML file and save it as ``docker-compose.yaml`` in the working directory.
+
+.. code-block:: yaml
+
+  services:
+    nuts:
+      image: nutsfoundation/nuts-node:latest
+      ports:
+        - 8080:8080
+        - 8081:8081
+      volumes:
+        - "./nuts.yaml:/nuts/nuts.yaml:ro"
+        - "./data:/nuts/data:rw"
+
+Start the service:
+
+.. code-block:: shell
+
+  docker compose up
+
+Docker ``run``
+^^^^^^^^^^^^^^
+
+If you want to run without Docker Compose you can use the following command from the working directory:
+
+.. code-block:: shell
+
+  docker run --name nuts -p 8080:8080 -p 8081:8081 \
+    --mount type=bind,source="$(pwd)",target=/nuts \
+    nutsfoundation/nuts-node:latest
```

Comment:
I would expect this one of the first things to see on this page.

Reply:
This

Reply:
do you want to swap `Docker run` and `Docker Compose`, or is it about moving the context (folder structure / nuts.yaml) further down